Vagrantfile

```diff
@@ -49,12 +49,12 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     end
 
     config.vm.provider "virtualbox" do |v|
-        v.memory = 1024
+        v.memory = 1536
```

Comment:
With `celery beat` and `celery worker` running, we quickly run out of memory on our vagrant deploys.